index.html

```diff
@@ -29,13 +29,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
         <title>CVSS v4.0 calculator</title>
         <script src="https://unpkg.com/vue@3.2.45/dist/vue.global.prod.js"></script>
-        <script src="cvss_config.js?v=2"></script>
-        <script src="cvss_details.js?v=1"></script>
-        <script src="cvss_lookup.js?v=5"></script>
-        <script src="cvss_score.js?v=2"></script>
-        <script src="max_composed.js?v=1"></script>
-        <script src="max_severity.js?v=1"></script>
-        <script src="metrics.js?v=1"></script>
+        <script src="cvss40.js"></script>
```

Comment:
Don't we still want some `?v=...` in there, to prepare for future evolution where we may have forgot that cache is a thing ? :cry:

Reply:
I didn't know why it was there in the first place, so I stupidly put it back here.

Reply:
To tell you the truth, I didn't know the point of this fake versioning. Which v should I use?

Reply:
I'm not really sure either, was this maybe a quick/hacky way to verify GH pages cache being broken? Additionally, the querystring doesn't seem to be used in any of the scripts.

Reply:
So, is this resolved?

Reply:
Maybe, I mimicked what was already done by @skontar, any tip or trick about this one ?

Reply:
Could I let you add the `v` when merged?